packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## NEXT
+## 1.1.12
 
 * Adds OS version support information to README.
+* Fixes `getAvailableBiometrics` behavior on Android. Now it returns an empty
+list when device has the biometric hardware but doesn't have any biometric
+enrolled. For more information check [this](https://github.com/flutter/flutter/issues/45497#issuecomment-1007310214).
```

Comment:
The last sentence can be removed; that comment doesn't really have any information that the CHANGELOG entry doesn't.